### PR TITLE
A few bug fixes in flaky-test-reporter

### PR DESCRIPTION
**Pull request by chaodaiG**
Fixing a few bugs with recent updates in flaky-test-reporter, this has been tested in dryrun mode and it worked

/cc @srinivashegde86

### CODE REVIEW

File: tools/flaky-test-reporter/github_issue.go [hunk 1]
```diff
@@ -551,10 +555,19 @@ func (gih *GithubIssueHandler) processGithubIssues(repoDataAll []RepoData, dryru
 
 	for _, rd := range repoDataAll {
 		issues, messages, err := gih.processGithubIssuesForRepo(rd, flakyGHIssuesMap, dryrun)
+		if _, ok := messagesMap[rd.Config.Repo]; !ok {
+			messagesMap[rd.Config.Repo] = make(map[string][]string)
```
Comment (srinivashegde86) [inline tools/flaky-test-reporter/github_issue.go]: is there an adv to using make? It's empty by default and will add it as the first item

Comment (chaodaiG) [reply tools/flaky-test-reporter/github_issue.go]: I see, thanks for the playground demo. actually this is commented on a wrong line, initializing map is still needed. I have removed the initialization of slices up and below

Comment (srinivashegde86) [reply tools/flaky-test-reporter/github_issue.go]: you dont need to check if a key exists to check the map init. We can use `len(messagesMap) == 0` or `messagesMap == nil` - https://play.golang.org/p/nNlgTAJan5r

**Pull request merged**